ui/src/pages/frontsite/StaticPage/StaticPage.js

```diff
@@ -0,0 +1,50 @@
+import { Error } from '@components';
+import {
+  ILSHeaderPlaceholder,
+  ILSParagraphPlaceholder,
+} from '@components/ILSPlaceholder';
+import { getStaticPageByRoute } from '@config/uiConfig';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Container, Header } from 'semantic-ui-react';
+import ReactHtmlParser from 'react-html-parser';
+import _isEmpty from 'lodash/isEmpty';
+
+export default class StaticPage extends Component {
+  componentDidMount() {
+    const staticPage = getStaticPageByRoute(this.props.match.path);
+    const staticPageID = staticPage['apiURL'];
+    this.props.fetchStaticPageDetails(staticPageID);
+  }
+
+  parseStaticPageContent = () => {
+    const { data } = this.props;
+    if (!_isEmpty(data)) {
+      // unescapes HTML and strips unsafe tags
+      const parsed = new DOMParser().parseFromString(data.content, 'text/html');
+      return ReactHtmlParser(parsed.body.innerText);
```

Comment:
Isn't this enough https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml instead of having another dependency?